Geometry/TrackerCommonData/plugins/dd4hep/DDTIDAxialCableAlgo.cc

```diff
@@ -0,0 +1,228 @@
+    Solid solid = ns.addSolid(
+        name, Polycone(-0.5 * width, width, pconRmin, pconRmax, pconZ));
+
+    LogVerbatim("TIDGeom") << "DDTIDAxialCableAlgo test: " << solid.name()
+                           << " Polycone made of " << matIn << " from "
+                           << convertRadToDeg(-0.5 * width) << " to "
+                           << convertRadToDeg(0.5 * width) << " and with "
+                           << pconZ.size() << " sections ";
+
+    for (int i = 0; i < (int)(pconZ.size()); i++)
+      LogVerbatim("TIDGeom")
+          << "\t[" << i << "]\tZ = " << pconZ[i] << "\tRmin = " << pconRmin[i]
+          << "\tRmax = " << pconRmax[i];
+
+    logs.emplace_back(
+        Volume(solid.name(), solid, ns.material("materials:Air")));
+  }
+
+  // Cable in the vertical part
+  std::vector<double> pconZ, pconRmin, pconRmax;
+  r = thktot * rMax / rTop;
+  z = zBend - thktot;
+  LogVerbatim("TIDGeom") << "DDTIDAxialCableAlgo test: Thk " << thk << " Total "
+                         << thktot << " rMax " << rMax << " rTop " << rTop
+                         << " dR " << r << " z " << z;
+
+  pconZ.emplace_back(z);
+  pconRmin.emplace_back(rMax);
+  pconRmax.emplace_back(rMax);
+
+  z = zBend - r;
+  pconZ.emplace_back(z);
+  pconRmin.emplace_back(rMax);
+  pconRmax.emplace_back(rTop);
+
+  pconZ.emplace_back(zBend);
+  pconRmin.emplace_back(rMax);
+  pconRmax.emplace_back(rTop);
+
+  std::string name = childName + std::to_string(zposWheel.size());
+
+  Solid solid = ns.addSolid(
+      name, Polycone(-0.5 * width, width, pconRmin, pconRmax, pconZ));
+
+  LogVerbatim("TIDGeom") << "DDTIDAxialCableAlgo test: " << solid.name()
+                         << " Polycone made of " << matIn << " from "
+                         << convertRadToDeg(-0.5 * width) << " to "
+                         << convertRadToDeg(0.5 * width) << " and with "
+                         << pconZ.size() << " sections ";
+
+  for (int i = 0; i < (int)(pconZ.size()); i++)
+    LogVerbatim("TIDGeom") << "\t[" << i << "]\tZ = " << pconZ[i]
+                           << "\tRmin = " << pconRmin[i]
+                           << "\tRmax = " << pconRmax[i];
+
+  logs.emplace_back(Volume(solid.name(), solid, ns.material(matIn)));
+
+  // Cable in the outer part
+  name = childName + std::to_string(zposWheel.size() + 1);
+  r = rTop - r;
+  solid = ns.addSolid(name,
+                      Tube(r, rTop, 0.5 * (zEnd - zBend), -0.5 * width, width));
+
+  LogVerbatim("TIDGeom") << "DDTIDAxialCableAlgo test: " << solid.name()
+                         << " Tubs made of " << matOut << " from "
+                         << convertRadToDeg(-0.5 * width) << " to "
+                         << convertRadToDeg(0.5 * width) << " with Rin " << r
+                         << " Rout " << rTop << " ZHalf "
+                         << 0.5 * (zEnd - zBend);
+
+  logs.emplace_back(Volume(solid.name(), solid, ns.material(matOut)));
+
+  // Position the cables
+  double theta = 90_deg;
```

Comment:
```suggestion
  constexpr double theta = 90_deg;
```

Reply:
@ianna @cvuosalo do you agree in updating this PR?

Reply:
@fabiocos: Looks good to me.